Puppetfile

```diff
@@ -1,7 +1,7 @@
 forge 'https://forgeapi.puppetlabs.com'
 
 mod 'atsonkov/grubby', '0.3.2'
-mod 'blockops/tailscale', git: 'https://github.com/lsst-it/puppet-tailscale', ref: 'd06b499'  # 1.0.0 + https://gitlab.com/blockops/puppet-tailscale/-/merge_requests/13
+mod 'blockops/tailscale', git: 'https://github.com/lsst-it/puppet-tailscale', ref: 'bc7e880'  # 1.0.0 + https://gitlab.com/blockops/puppet-tailscale/-/merge_requests/13
```

Comment:
The comment is no longer correct. https://gitlab.com/blockops/puppet-tailscale/-/merge_requests/13 was merged upstream and there is not description of the changes in the fork (linking a PR would be ideal).